### PR TITLE
Py3convert

### PR DESCRIPTION
**Pull request by JSchwerberg**
## Short Description:
Update bot to be cross compatible with Python 3.5+ and Python 2.7+.  

Was already being tested at
https://github.com/PokemonGoF/PokemonGo-Bot-Backup/pull/26

## Fixes/Resolves/Closes (please use correct syntax):
N/A

### CODE REVIEW

**Comment by javajohnHub** [comment]
I tested on 2.7 for 2 hours with no issues.

File: pokemongo_bot/cell_workers/transfer_pokemon.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-import os
 
 from pokemongo_bot import inventory
 from pokemongo_bot.human_behaviour import action_delay
 from pokemongo_bot.base_task import BaseTask
-from pokemongo_bot.inventory import Pokemons, Pokemon, Attack
+from pokemongo_bot.inventory import Attack
+from pokemongo_bot.inventory import Pokemon
+from pokemongo_bot.inventory import Pokemons
 from operator import attrgetter
```
Comment (alexyaoyang) [inline pokemongo_bot/cell_workers/transfer_pokemon.py]: Just curious over here, is there a reason for doing this?

Comment (sohje) [inline pokemongo_bot/cell_workers/transfer_pokemon.py]: pep-8

Comment (alexyaoyang) [inline pokemongo_bot/cell_workers/transfer_pokemon.py]: Ah okay good 👍

Comment (JSchwerberg) [inline pokemongo_bot/cell_workers/transfer_pokemon.py]: Yeah, I ran `flake8` over the codebase during the conversion in e31c2a7~ -- figured it was a good time to make those changes, while I was mass-changing imports anyway. :)

**Comment by JSchwerberg** [comment]
GitHub refuses to update with the proper commits, fixing that merge conflict.  Closing and submitting a new PR.

**Pull request closed without merging**